packages/utils/upgrade/resources/transforms/5.0.0/upgrade-strapi-version.json.ts

```diff
@@ -6,22 +6,24 @@ import type { JSONTransform } from '../../..';
  * Note: This transform file is only for development purposes and should be deleted before releasing
  */
 
-const transform: JSONTransform = (file, api) => {
-  const packageJsonPath = path.join(api.cwd, 'package.json');
+const transform: JSONTransform = (file, params) => {
+  const { cwd, json } = params;
 
   // Ignore files that are not the root package.json
   // Note: We could also find every file named package.json and update the dependencies for all of them
-  if (file.path !== packageJsonPath) {
+  const rootPackageJsonPath = path.join(cwd, 'package.json');
+  if (file.path !== rootPackageJsonPath) {
     return file.source;
```

Comment:
I would stick with root or source in both places, when I see file.source my first thought is just a raw string of the entire file and I was confused at first about why we were returning a file here and an object down below.

Or honestly something even more explicit, like `parsedJson` `json` `jsonValue` `jsonObject`, not sure, but if you have an idea :)

Reply:
I agree, it can be confusing. I went with `file.json`